### PR TITLE
Eric Jankowski's instructor training pull request

### PR DESCRIPTION
**Pull request by erjank**
Fixed typos and misspellings in main markdown and episode markdown files.

### CODE REVIEW

File: _episodes/02-filedir.md [hunk 1]
```diff
@@ -273,7 +273,7 @@ Mandatory arguments to long options are mandatory for short options too.
                                file-type (--file-type), classify (-F)
   -i, --inode                print the index number of each file
   -I, --ignore=PATTERN       do not list implied entries matching shell PATTERN
-  -k, --kibibytes            default to 1024-byte blocks for disk usage
+  -k, --kilobytes            default to 1024-byte blocks for disk usage
```
Comment (gdevenyi) [inline _episodes/02-filedir.md]: Hi, this isn't a typo. The manpage says this, and this is a different unit: https://en.wikipedia.org/wiki/Kibibyte

Can you please update the PR to remove this.

File: _episodes/05-loop.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ rather than treat it as text or an external command.
 In this example, the list is two filenames: `basilisk.dat` and `unicorn.dat`.
 Each time the loop iterates, it will assign a file name to the variable `filename`
 and run the `head` command.
-The first time throught the loop,
+The first time through the loop,
```
Comment (gdevenyi) [inline _episodes/05-loop.md]: This was fixed by another user, can you please rebase the PR. Thanks.

**Comment by erjank** [comment]
Hi Gabe, I believe I've now correctly made the requested kibibytes and rebasing fix. Let me know if it looks good or if there's additional action required. Thanks!

**Comment by gdevenyi** [comment]
Wonderful! Thanks for the fixes!

**Pull request merged**